### ISSUE TITLE
Our tests are breaking because of error thrown in `casper-js-sdk`

### ISSUE THREAD

**Issue by piotrwitek** [casper-ecosystem/casper-client-sdk#323]
**Describe the bug**
Our tests are breaking because of an error thrown in `casper-js-sdk`.
This will probably block our release schedule, we'll discuss this with the team, meanwhile please suggest a solution.

```
● Console

    console.error
      @jsonMember on e.StoredValue: cannot resolve specified property constructor at runtime. If the type is not yet defined, for example due to circular references, add '() => ' before it. E.g. @jsonMember(() => Foo)

    > 1 | import { DeployUtil, Keys } from 'casper-js-sdk';
        | ^
      2 | import { signDeploy } from './sign-deploy';
      3 |
      4 | const getSignature = (

      at Object.logError (node_modules/typedjson/src/helpers.ts:83:22)
      at node_modules/typedjson/src/json-member.ts:157:17
      at BalanceServiceByJsonRPC (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/src/services/BalanceServiceByJsonRPC.ts:8:14)
      at node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/src/services/CasperServiceByJsonRPC.ts:235:3
      at Object.2969 (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/src/services/CasperServiceByJsonRPC.ts:225:1)
      at call (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/webpack/bootstrap:19:32)
      at Object.4400 (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/src/services/index.ts:1:1)
      at call (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/webpack/bootstrap:19:32)
      at Object.7108 (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/src/index.ts:1:1)
      at call (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/webpack/bootstrap:19:32)
      at __webpack_require__ (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/webpack/startup:4:27)
      at node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/webpack/startup:4:47
      at factory (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/webpack/universalModuleDefinition:3:20)
      at Object.<anonymous> (node_modules/casper-js-sdk/dist/webpack:/casper-js-sdk/webpack/universalModuleDefinition:1:1)
      at Object.<anonymous> (src/libs/crypto/sign-deploy.test.ts:1:1)
```

**To Reproduce**

Identify if any Prequisites:

1. `import { DeployUtil, Keys } from 'casper-js-sdk';`
2. write some simple test that is using above as dependencies
3. run a test with nodejs
4. Tests are crashing with error

**Expected behavior**
No error

**Screenshots**

**Additional context**

**Defect Type**
- [x] Regression
- [ ] New
- [ ] Escape

**Comment by gyroflaw** [comment]
Fixed in https://github.com/casper-ecosystem/casper-js-sdk/pull/312

**Comment by KillianH** [comment]
@gyroflaw the fix isn't released yet. It's still blocking most of web devs

**Comment by gyroflaw** [comment]
this should be released in the next 2.14 version

**Comment by gyroflaw** [comment]
closing this as 2.14 released
